config.php-RECOMMENDED

```diff
@@ -25,9 +25,9 @@
     -----------------------------
     If installation is in a eZ Publish 5/Symfony context, composer will be attempted to be used
     if the following file is found: '../EZP_APP_FOLDER_NAME/autoload.php'
-    Default EZP_APP_FOLDER_NAME value is "ezpublish" and this constant gives you possibility to change it.
+    Default EZP_APP_FOLDER_NAME value is "app" and this constant gives you possibility to change it.
```

Comment:
Well, it's using a tagged version of eZ Publish legacy, so no immediate breakage.